tests/react-a11y-mouse-event-has-key-event/test.tsx.lint

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+
+const element = (<div onMouseOver={() => {}}>click</div>)
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~             [onMouseOver must be accompanied by onFocus.]
+const elementTwo = (<div onMouseOut={() => {}}>click</div>)
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~             [onMouseOut must be accompanied by onBlur.]
+const elementSelfClosing = (<div onMouseOut={() => {}}/>)
+                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~  [onMouseOut must be accompanied by onBlur.]
+
+<div onMouseOver={() => {}} {...props} />
+<div onMouseOut={() => {}} {...props} />
+<div />
+<div {...props} />
```

Comment:
These four cases, where the line starts with `<`, are invalid TypeScript. You'll need to store these in a variable as with the other cases.

Reply:
Oh oops.
Ok, so
```
const foo = <div onMouseOver={() => {}} {...props} />
const bar = <div onMouseOut={() => {}} {...props} />
``` 
does complain, which makes sense. So do you just ignore elements with spread attributes?

Reply:
I think so, yeah. Without a program's type checker there's no way to know if they're valid.